### ISSUE TITLE
[A11y] Explore Collection List: Find A Collection Search

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1926]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Explore Collections

### Admin user?

No

### Issue Description

Explore Collection List: https://staging.oregondigital.org/explore_collections/all?locale=en

"Find a Collection" search box has the following failed instance:
duplicate-id-aria: Ensures every id attribute value used in ARIA and in labels is unique

Path: 
form[action="\/explore_collections\/all"]

Snippet:
`<form class="form-horizontal search-form" id="explore-collection-search" role="search" action="/explore_collections/all" accept-charset="UTF-8" method="get">`

### Suggested Fix

Fix the following:

- Document has multiple elements referenced with ARIA with the same id attribute: explore-collection-search

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by CGillen** [comment]
Fixed by #1965
